src/platforms/ctxlink/WiFi_Server.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+void app_initialize(void);
+void app_task(void);
+//
+void gdb_tcp_server(void);
+bool is_gdb_client_connected(void);
+
+void data_tcp_server(void);
+bool is_uart_client_connected(void);
+void send_uart_data(uint8_t *lpBuffer, uint8_t length);
```

Comment:
`buffer` instead of `lpBuffer` please. Same for `send_swo_trace_data()`.

Reply:
Changed as requested.